go/vt/vttablet/tabletserver/repltracker/writer.go

```diff
@@ -182,6 +187,9 @@ func (w *heartbeatWriter) recordError(err error) {
 
 // enableWrites actives or deactives heartbeat writes
 func (w *heartbeatWriter) enableWrites(enable bool) {
+	if w.ticks == nil {
+		return
+	}
```

Comment:
@deepthi this change is actually reverting my previous change in the throttler PR. This `if` statement was originally here.